dss/events/handlers/sync.py

```diff
@@ -1,20 +1,35 @@
+import os
+import sys
 import datetime
+import json
 import time
 import hashlib
 from contextlib import closing
+from concurrent.futures import ThreadPoolExecutor
+from collections import namedtuple
 
 import boto3
 import botocore.session
 import urllib3
 import google.cloud.storage
+from google.resumable_media._upload import get_content_range
 from google.cloud.client import ClientWithProject
 from google.cloud._http import JSONConnection
 
 from dss import Config
+from dss.util.aws import resources, clients, send_sns_msg, ARN
+from dss.util.streaming import get_pool_manager, S3SigningChunker
+from dss.blobstore.gs import GSBlobStore
 
 presigned_url_lifetime_seconds = 3600
 use_gsts = False
 gsts_sched_delay_minutes = 2
+part_size = {"s3": 64 * 1024 * 1024, "gs": 64 * 1024 * 1024 * 10}
```

Comment:
The rationale for the part_size for s3 is obvious.
What is the rationale for the part_size of gs?
How does this guarantee that we stay under the component count limit?

I would prefer some scheme where we figure out what the part size is to accommodate the component, and then apply some rational floor to the part size.  I think it's reasonable to do this in a followup.

Also, I find 640 * 1024 * 1024 easier to read.